Makefile

```diff
@@ -0,0 +1,198 @@
+# Copyright (C) 2024, The Valkey contributors
+# All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+
+# Path to the code repo.
+VALKEY_ROOT ?= ../valkey
+
+# Where to install man pages
+INSTALL_MAN_DIR ?= /usr/local/share/man
+
+# Where to put the generated pages.
+BUILD_DIR ?= _build
+MD_DIR ?= $(BUILD_DIR)/md
+MAN_DIR ?= $(BUILD_DIR)/man
+HTML_DIR ?= $(BUILD_DIR)/html
+
+# -----------------------------------
+
+.PHONY: all man html md clean distclean install uninstall
+
+all: man
+
+# ---- Sanity check ----
```

Comment:
Add check for pandoc which might not be that commonly installed?
i.e. a short text as an alternative to:
```
> make
mkdir -p _build/man/man1
utils/preprocess-markdown.py --man --page-type program topics/benchmark.md \
 | utils/links-to-man.py - | pandoc -s --to man -o _build/man/man1/valkey-benchmark.1 -
/bin/sh: 2: pandoc: not found
Traceback (most recent call last):
  File "/home/bjorn/repos/valkey-doc-orig/utils/links-to-man.py", line 129, in <module>
    print(content)
BrokenPipeError: [Errno 32] Broken pipe
make: *** [Makefile:142: _build/man/man1/valkey-benchmark.1] Error 127
```